### DISCUSSION TITLE
AMD plots Very slow

### DISCUSSION THREAD

**Discussion by Kingson1874** [swar/Swar-Chia-Plot-Manager#801] [Q&A]
I use and 5900x and found that plots are very slow.

**Comment by vviLab** [comment]
Dude, Please give more detail of your hardware, your configuration...

**Reply by BiAn-kisstheme** [reply]
设备：
SSD：M.2 NVMe SSD 1T * 4（不受欢迎的品牌）

内存：64G（32g 3600Ghz * 2）

CPU：AMD锐龙9 5900X（12C 24T）

主板：技嘉X570 GAMING X

**Reply by Kingson1874** [reply]
Amd 5950x
DDR4 3200 128G
Nvme 2T x2

**Comment by andy118z** [comment]
should fast with this processor

**Reply by Kingson1874** [reply]
Yes

**Comment by PPMNUTRA** [comment]
Guys my Specs is:

Ryzen ThreadRipper 3960 wx
128 GB RAM 3400 MHZ
2 SSD SN750 2 TB
1 SSD XPF SX8100 4TB
MB AORUS EXTREME TR4


I need know, how much plots i can do per day and paralell, someone can help me

**Comment by creamsoup** [comment]
INTEL plots Even slower

**Comment by creamsoup** [comment]
we can't help you from nothing.
don't blame system. it is usually user error. 

what's the definition of slow? any measurement? what's expected value of your system's throughput or whatever metric you are using? and what is the observed number? what's your swar configuration? system configuration? os?

**Comment by DomChia** [comment]
I have the same problem.
AMD 5900x
32GB ram 3600
2x 2TB Samsung 980 pro   RAID0
25 plots per day...

global ->
  max_concurrent: 14
  max_for_phase_1: 5
  minimum_minutes_between_jobs: 5

Jobs ->
    size: 32
    bitfield: true
    threads: 6
    buckets: 128
    memory_buffer: 3400
    max_concurrent: 12
    max_concurrent_with_start_early: 14
    initial_delay_minutes: 0
    stagger_minutes: 35
    max_for_phase_1: 5
    concurrency_start_early_phase: 5
    concurrency_start_early_phase_delay: 0
    temporary2_destination_sync: false
    exclude_final_directory: false
    skip_full_destinations: true
    unix_process_priority: 10
    windows_process_priority: 32
    enable_cpu_affinity: false
    cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]

**Reply by creamsoup** [reply]
try 4 threads. disable raid0. 6ish concurrent, 30min stagger, 3 phase 1 for  each ssd.
my 5900x is easily getting 30+ upto 35 with similar hardware specs. (5950x is my main so i haven't spend time to really optimize 5900x)

**Reply by mbabak74** [reply]
check what is the bottleneck. Is you cpu maxed out? Is your drive maxed out?
When you know which one, you will know what to fix. Probably you have to many jobs running at the same time for your drive to handle..
